openfisca_us/parameters/states/ny/tax/income/credits/nonrefundable/household_credit/non_single/additional.yaml

```diff
@@ -0,0 +1,28 @@
+description: New York State household credit additional amount per person for non-single filers.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: currency-USD
+  label: NY household credit additional amount for non-single filers
+  reference:
+    - title: Instructions for Form IT-203
+      href: https://www.tax.ny.gov/pdf/current_forms/it/it203i.pdf#page=28
+    - title: Section 606 Credits against tax (b)(2)(B)
+      href: https://www.nysenate.gov/legislation/laws/TAX/606
+brackets:
+  - threshold:
+      1986-01-01: -.inf
+    amount:
```

Comment:
Very minor, I think if we include `amount` here then `type: single_amount` is inferred by OpenFisca. No need to change though.